modules/nf-core/sentieon/bwaindex/main.nf

```diff
@@ -0,0 +1,70 @@
+process SENTIEON_BWAINDEX {
+    tag "$fasta"
+    label 'process_high'
+    label 'sentieon'
+
+    secret 'SENTIEON_LICENSE_BASE64'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Sentieon modules does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container 'nfcore/sentieon:202112.06'
```

Comment:
What container-repo is that? Is it something new made especially for sentieon?

Reply:
Just a dump from https://github.com/Sentieon/sentieon-docker
Made locally and pushed to the nfcore dockerhub space